src/components/pages/blogs/blog/components/ListTemplate.tsx

```
@@ -1,45 +1,115 @@
+                        <h3 className="text-lg text-gray-600 font-semibold">{content.subTitle}</h3>
+                    )}
+                </div>
+            </div>
+
+            {/* Top Description */}
+            {content.topDescription && (
+                <div className="mb-8">
+                    {content.topDescription.split("\n").map((paragraph, index) => (
+                        <p className="text-gray-700 leading-relaxed mb-4 text-base lg:text-lg"
+                            key={index}
+                            dangerouslySetInnerHTML={{ __html: paragraph }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)